.github/workflows/docker-build-and-push.yml

```
@@ -96,3 +96,7 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
           tags: ${{ env.TAGS }}
+          build-args: |
```

actionlint (1.7.4)

format string "{0:yyyy-MM-ddTHH:mm:ssZ}" does not contain placeholder {0}. remove argument which is unused in the format string

(expression)